src/components/PostPageDetail/elements/Content.js

```diff
@@ -249,8 +251,7 @@ const Content = ({
               item
             })}
           </View>
-        )}
-        {images_url?.length > 0 && (
+        ) : (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering based on the presence of images in the `images_url` array is a good practice. This ensures that the image layouter is only used when there are images to display, improving performance.
```diff
-        {images_url?.length > 0 && (
+        ) : (
           <View style={styles.containerImage}>
             <ImageLayouter
               mode={FastImage.resizeMode.cover}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->